src/main.py

```diff
@@ -305,10 +315,17 @@ async def main() -> None:
                             )
                             knative_serving_url = latiss_knative_serving_url
                             in_process_requests_gauge = latiss_in_process_requests_gauge
-                        # case "LSSTComCam":
-                        #    fan_out_message_list = next_visit_message.add_detectors(
-                        #        next_visit_message, lsst_com_cam_active_detectors
-                        #    )
+                        case 3:  # LSSTComCamSim
```

Comment:
I assume that if the Summit scheduler does something different, we'll adjust the dev system to match?

Reply:
Yes, that will be [DM-42713](https://jira.lsstcorp.org/browse/DM-42713).